README.md

```diff
@@ -37,12 +28,10 @@ yetAnotherLocale =
 formatFloat : Locale -> Float -> String
 ```
 
-Format a `Float` number as a pretty string. For example, using the locales defined above:
+Format a `Float` number as a pretty string. For example, using the locale defined above:
 
 ```elm
 formatFloat defaultLocale 1234.5567 == "1,234.56"
-formatFloat alternativeLocale -7654.3210 == "-7.654,321"
```

Comment:
I wanted to make the README a very short description, that gives an quick overview of the project, and not a documentation.

The full documentation is inside the module. I actually added a lot of edge cases to the documentation.